### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by jasonzuoj** [aglarcz/WotlkClassic-Kaliel-s-Tracker#1]
1x !KalielsTracker\Modules\QuestLog.lua:145: attempt to call global 'QuestWatch_Update' (a nil value)
[string "@!KalielsTracker\Modules\QuestLog.lua"]:145: in function <!KalielsTracker\Modules\QuestLog.lua:137>
[string "=[C]"]: ?
[string "@Ace3\AceAddon-3.0\AceAddon-3.0-13.lua"]:70: in function <Ace3\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Ace3\AceAddon-3.0\AceAddon-3.0-13.lua"]:527: in function <Ace3\AceAddon-3.0\AceAddon-3.0.lua:520>
[string "=(tail call)"]: ?
[string "@!KalielsTracker\KalielsTracker.lua"]:1729: in function <!KalielsTracker\KalielsTracker.lua:1720>
[string "=[C]"]: ?
[string "@Ace3\AceAddon-3.0\AceAddon-3.0-13.lua"]:70: in function <Ace3\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Ace3\AceAddon-3.0\AceAddon-3.0-13.lua"]:527: in function `EnableAddon'
[string "@Ace3\AceAddon-3.0\AceAddon-3.0-13.lua"]:630: in function <Ace3\AceAddon-3.0\AceAddon-3.0.lua:615>

**Comment by thelomere** [comment]
Same here

2x !KalielsTracker\Modules\QuestLog.lua:145: attempt to call global 'QuestWatch_Update' (a nil value)
[string "@!KalielsTracker\Modules\QuestLog.lua"]:145: in function <!KalielsTracker\Modules\QuestLog.lua:137>
[string "=[C]"]: ?
[string "@Questie\Libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:70: in function <...ceQuestie\Libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Questie\Libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:527: in function <...ceQuestie\Libs\AceAddon-3.0\AceAddon-3.0.lua:520>
[string "=(tail call)"]: ?
[string "@!KalielsTracker\KalielsTracker.lua"]:1729: in function <!KalielsTracker\KalielsTracker.lua:1720>
[string "=[C]"]: ?
[string "@Questie\Libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:70: in function <...ceQuestie\Libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Questie\Libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:527: in function `EnableAddon'
[string "@Questie\Libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:630: in function <...ceQuestie\Libs\AceAddon-3.0\AceAddon-3.0.lua:615>

**Comment by Fr1tzl1** [comment]
Message: ...nterface\AddOns\!KalielsTracker\Modules\QuestLog.lua:145: attempt to call global 'QuestWatch_Update' (a nil value)
Time: Wed Sep  7 20:28:04 2022
Count: 1
Stack: ...nterface\AddOns\!KalielsTracker\Modules\QuestLog.lua:145: attempt to call global 'QuestWatch_Update' (a nil value)
[string "@Interface\AddOns\!KalielsTracker\Modules\QuestLog.lua"]:145: in function <...nterface\AddOns\!KalielsTracker\Modules\QuestLog.lua:137>
[string "=[C]"]: ?
[string "@Interface\AddOns\Questie\Libs\AceAddon-3.0\AceAddon-3.0.lua"]:70: in function <...ce\AddOns\Questie\Libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Interface\AddOns\Questie\Libs\AceAddon-3.0\AceAddon-3.0.lua"]:527: in function <...ce\AddOns\Questie\Libs\AceAddon-3.0\AceAddon-3.0.lua:520>
[string "=(tail call)"]: ?
[string "@Interface\AddOns\!KalielsTracker\KalielsTracker.lua"]:1729: in function <Interface\AddOns\!KalielsTracker\KalielsTracker.lua:1720>
[string "=[C]"]: ?
[string "@Interface\AddOns\Questie\Libs\AceAddon-3.0\AceAddon-3.0.lua"]:70: in function <...ce\AddOns\Questie\Libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Interface\AddOns\Questie\Libs\AceAddon-3.0\AceAddon-3.0.lua"]:527: in function `EnableAddon'
[string "@Interface\AddOns\Questie\Libs\AceAddon-3.0\AceAddon-3.0.lua"]:630: in function <...ce\AddOns\Questie\Libs\AceAddon-3.0\AceAddon-3.0.lua:615>
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:386: in function `UIParentLoadAddOn'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:444: in function `RaidFrame_LoadUI'
[string "@Interface\FrameXML\RaidFrame.lua"]:55: in function <Interface\FrameXML\RaidFrame.lua:50>

**Comment by aglarcz** [comment]
When this happens? Did you change everything from default settings?

**Comment by thelomere** [comment]
Nope, only copied in the interface folder, activate in the addon UI and started the game. Bugsack catches the error

**Comment by aglarcz** [comment]
Which version of questie are you using?

**Comment by thelomere** [comment]
Latest stable, 7.0.2 from Curseforge

**Comment by aglarcz** [comment]
@thelomere well i see the error now via bugsack but tracker is working fine anyway.. can you try to click F (filter) and set it to autozone if thats works for you?

**Comment by thelomere** [comment]
nope, F is green, autofilter activated. Tracker is empty with marked quests in the log

**Comment by aglarcz** [comment]
okay so i get the issue now - tracker is working but just the part tracking via zones, manual tracking via questlog is broken.. 
can you please get into some zone with quests you got and confirm this for me?
